src/route/index.ts

```diff
@@ -37,6 +42,41 @@ export function initApiServer(mercuryClient: MercuryClient) {
         },
       });
 
+      instance.route({
+        method: "GET",
+        url: "/account-balances/:pub-key",
+        schema: {
+          params: {
+            ["pub-key"]: { type: "string" },
```

Comment:
no need to make this optimization now, but might be useful to think about if we want to store things like params and query string values in an enum as we introduce more routes/complexity